yarn-error.log

```diff
@@ -1,38 +1,36 @@
 Arguments: 
```

Comment:
can we remove and gitignore this file?

Reply:
Yup, looks like there are some others too

Reply:
https://github.com/Wikia/jwplayer-fandom/pull/82/commits/c7431adf5dc15f6716ac3cf565bc276c9e9afb5d